sir_complainsalot/delphi_sir_complainsalot/check_source.py

```diff
@@ -63,8 +63,8 @@ def check_source(data_source, meta, params, grace, logger):
     gap_complaints = {}
 
     for _, row in signals.iterrows():
-        if "retired-signals" in source_config and \
-           row["signal"] in source_config["retired-signals"]:
+        retired_signals = source_config.get("retired-signals")
```

Comment:
probably should move this outside the loop -- there are like 600 rows in meta for some of the sources